app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -373,6 +372,14 @@ class BrowserTabViewModel(
             if (shouldClearHistoryOnNewQuery()) {
                 command.value = ResetHistory
             }
+
+            val queryChanged = if (currentOmnibarViewState().omnibarText == query){
```

Comment:
this pixel is useful during the experiment not only for the header removal variant, but also to compare engagement between both search boxes in the control group and other variant